Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/Survey.kt

```diff
@@ -69,6 +71,11 @@ sealed class PossibleAnswer {
         @StringRes val neutralText: Int,
         val defaultValue: Float = 5.5f
     ) : PossibleAnswer()
+
+    data class Group(
```

Comment:
Consider calling this "Grouping" to be clearer that this is not indicating a single group. 

Also, this might be a good place to add a comment explaining what this type of question is, something like: "Asks user to divide the [items] up between the various [groups]."

I know the other PossibleAnswer types don't have documentation yet, but this might set a good trend :)

Reply:
Thanks :) Added in the next commit